### PR TITLE
Homework #1

### PR DESCRIPTION
**Pull request by kuskovars**
Create simple alert block that can be called from say(name) function

### CODE REVIEW

File: task_1/dist/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+main.js
```
[the remark this answers is not in the document]
Comment (kuskovars) [reply task_1/dist/.gitignore]: Fixed

File: task_1/dist/css/style.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    position: absolute;
+    background-color: #000;
+    color: #fff;
+    width: 50%;
+    height: 10em;
+    border: 3px solid grey;
+    padding: 10px;
+    text-align: center;
+    display: none;
+}
```
[the remark this answers is not in the document]
Comment (kuskovars) [reply task_1/dist/css/style.css]: Fixed

File: task_1/src/lib/test.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const say = function(name)
+{
+	document.getElementById("modal").innerHTML = 'Hello, ' + name +'!';
```
[the remark this answers is not in the document]
Comment (kuskovars) [reply task_1/src/lib/test.js]: Fixed

**Comment by martinkomitsky** [comment]
супер, 10

**Pull request merged**